austrian-journal-of-development-studies.csl

```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" name-delimiter="/" demote-non-dropping-particle="never" default-locale="de-AT">
```

Comment:
@POBrien333, if this is a polyglot CSL style it shouldn't have a default-locale, right?

Reply:
(or did you only develop/test this style for German?)

Reply:
thanks. An oversight. I've removed the default-locale.